components/abracon/AMPM.stanza

```diff
@@ -50,36 +53,8 @@ public pcb-component component (frequency:Double):
     50.0e6   : ["AMPMAFB-50.0000T"  25.0e-6  VFLGA-2 ]    
     8.0e6    : ["AMPMEFB-8.0000T"   25.0e-6  VDFN    ]  
     80.0e6   : ["AMPMAFB-80.0000T"  25.0e-6  VFLGA-2 ]    
-
-    ; 10.0e6   : ["AMPMGFB-10.0000T"  25.0e-6  VFLGA-16]      
```

Comment:
These lines, although commented out, still contain useful information. Can we keep them?

Reply:
How is the information intended to be used? I'm hesitant to keep large chunks of code that isn't functional. We can always add it back later.

Reply:
It's used to model the rest of the devices in this family. The component would get an extra argument for package, and a few more landpattern options would be added. I like having breadcrumbs that make it easy to keep adding to ocdb but if you find it disruptive, it's ok to delete it.